server/middlewares/auth/index.js

```diff
@@ -0,0 +1,20 @@
+const Boom = require('@hapi/boom');
+const { verifyToken } = require('../../utils');
+
+const Auth = async (req, res, next) => {
+  const { token } = req.cookies;
+  if (token) {
+    try {
+      req.user = await verifyToken(token);
+      next();
+    } catch (err) {
+      throw Boom.unauthorized('You are not Authorized');
```

Comment:
Inside  Auth handler, you should put outer try {}catch(err){next(err)
, to catch this error in the server handler

Reply:
Ok :+1: